### PR TITLE
Add setup script and template files

### PR DESCRIPTION
**Pull request by ewquon**
Files courtesy of @phawbeck.

Changes:
- Remove `$HOME_DIR` parameter, instead storing template files in a subdirectory
- Remove hard-coded email/passwords
- Add prompt for email/pass

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -27,37 +28,55 @@ ddS="08"
 hhS="00"
 # NUMBER OF DOMAINS TO USE
 MAX_DOM="2"
+# DOMAIN CONFIG (default: "")
+DOM_STR=""
```
Comment (hawbecker) [inline setup.sh:32]: When would this be needed?

File: setup.sh [hunk 2]
```diff
@@ -80,23 +106,23 @@ do
...
     if [ $REAN_STR == "ERA" ]; then
-        sed "s/SETPASSWD/nrel1234/g" $HOME_DIR/get_erai_template.csh > get_erai1.csh
-        sed "s/DD1/$dd/g" get_erai1.csh > get_erai2.csh && rm get_erai1.csh 
-        sed "s/MM1/$mm/g" get_erai2.csh > get_erai3.csh && rm get_erai2.csh 
-        sed "s/YY1/$yy/g" get_erai3.csh > get_erai.csh && rm get_erai3.csh 
-        chmod 777 get_erai.csh
-        ./get_erai.csh
-        cp get_erai.csh get_erai.csh_$CASE_STR"_"$counter
+        sed "s/SETPASSWD/$passwd/g" $TEMPLATE_DIR/get_erai_template.csh > get_erai.csh
+        sed -i "s/MY.EMAIL@asdf.com/$emailaddr/g" get_erai.csh
```
Comment (hawbecker) [inline setup.sh:104]: I don't know how when I was searching for "how to use sed" this never came up. Everything I saw was saying you should save to a new file name every time... this is much better.

File: setup.sh [hunk 3]
```diff
@@ -124,9 +150,16 @@ echo "Finished downloading ICBC data."
 #                               Run WPS               
 #======================================================================
 #======================================================================
+
+WPS_TEMPLATE="$TEMPLATE_DIR/namelist.wps.template"
+if [ -n "$DOM_STR" ]; then
+    # append DOM_STR
+    WPS_TEMPLATE="${WPS_TEMPLATE}.${DOM_STR}"
```
Comment (hawbecker) [inline setup.sh:152]: Typically you'll only be running WPS once for all domains.

**Review by hawbecker — COMMENTED** [review]
Left a few comments on the DOM_STR - just not sure what this would be used for.

File: setup.sh (hunk 1 above)
Comment (ewquon) [reply setup.sh:32]: To select `namelist.wps.template*`

**Review by hawbecker — APPROVED** [review]
I understand the "DOM_STR" variable now to be what selects something like "namelist.wps.template.large" or any other variable namelist template.

**Review by twjuliano — APPROVED** [review]
I think we need the following command somewhere for the WRF run:

`cp myoutfields.txt $OUT_DIR
`

Otherwise, namelist.input points to a txt file that is not in the newly-created folder.

**Comment by ewquon** [comment]
I've fixed the issue brought up by @twjuliano. Also, I changed the workflow a bit, to something more in line with what I had originally imagined. 

My original intention was to track namelist.input (and any auxiliary WRF input files); the namelist.wps and other files should be straightforward to generate manually for an experienced WRF user. However, after adding the nice `setup.sh` script that @phawbeck provided, I realized that all generated input files would be placed outside of the repository and therefore not be tracked (per my original intention). The new workflow follows what I had originally outlined in the README.

**Pull request merged**